doc/source/notebooks/advanced/coregionalisation.pct.py

```diff
@@ -136,14 +136,14 @@ def plot_gp(x, mu, var, color, label):
 
 def plot(m):
     plt.figure(figsize=(8, 4))
-    xtest = np.linspace(0, 1, 100)[:, None]
+    Xtest = np.linspace(0, 1, 100)[:, None]
```

Comment:
I get why you added these to the PR, but I'd much prefer it if they were in a separate PR. it's not necessarily quick to work out what should be `x` and `X`

Reply:
would you mind splitting this into two PRs? one for just predict method parameters and one for everything else?